app/src/main/java/org/projectbuendia/client/ui/dialogs/EditPatientDialogFragment.java

```diff
@@ -56,6 +62,8 @@
     @InjectView(R.id.patient_sex_female) RadioButton mSexFemale;
     @InjectView(R.id.patient_sex_male) RadioButton mSexMale;
 
+    private static final Pattern ID_PATTERN = Pattern.compile("([a-zA-Z]+)/?([0-9]+)*");
```

Comment:
I added the * in the end to match blank ids. This fixes the problem when editing the patient.
@dancunningham This fix the previous bug.